scripts/src/main/resources/scripts/migration

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env bash
+
+# shellcheck source=scripts/functions
+source "$(dirname "${0}")"/functions
+
+from_version=2023.02.001
+if [ -f "${DEVON_IDE_HOME}/.devon.software.version" ]  
+then
+  from_version="$(cat ${DEVON_IDE_HOME}/.devon.software.version)"
+fi
+
+MIGRATIONS_DIR="${DEVON_IDE_HOME}/scripts/migrations"
+
+if [ -d "${MIGRATIONS_DIR}" ]
+then
+  cd "${MIGRATIONS_DIR}"
+  for migrate_to_version in  $(ls | sort -n)
+  do
+    doVersionCompare "${from_version}" "${migrate_to_version}"
+    result="${?}"
+    if [ "${result}" -eq 0 ] || [ "${result}" -eq 2 ]
```

Comment:
after rethinking this, we should only migrate if `${from_version}` is less `${migrate_to_version}` and NOT if equal (see previous review comment).
If we agree and change this accordingly, we need to change `from_version=2023.02.001` to `from_version=2023.01.001`.
If I am not missing something and thought everything through correctly, all should be working perfectly then.

Reply:
Should work now

Reply:
Thanks.